gui/default/index.html

```diff
@@ -768,6 +768,13 @@ <h4 class="panel-title">
                       <th><span class="fas fa-fw fa-folder"></span>&nbsp;<span translate>Folders</span></th>
                       <td class="text-right" ng-attr-title="{{deviceFolders(deviceCfg).map(folderLabel).join(', ')}}">{{deviceFolders(deviceCfg).map(folderLabel).join(", ")}}</td>
                     </tr>
+                    <tr ng-if="findDevice(deviceCfg.deviceID).remoteGUIPort > 0">
```

Comment:
`findDevice()` was removed on the main branch.  Just use the `deviceCfg` object itself:
```suggestion
                    <tr ng-if="deviceCfg.remoteGUIPort > 0">
```

Reply:
This still needs fixing in the latest commit 6847411b.

Reply:
idk why this is not updating, but I change it to your suggestion. You can even see it in my commit https://github.com/syncthing/syncthing/pull/7017/commits/ed3cd03cab1054026b10010a559f1985618fded1. It seems unchanged on the view file changes.

Reply:
You need to be careful for which commit GitHub is showing the diff.  It is indeed fixed now.  Feel free to mark this conversation "resolved" (seems I can't do that although I started it myself).